AUTHORS.md

```diff
@@ -56,6 +56,9 @@ To contributors: please add your name to the list when you submit a patch to the
    * SAR PySpark improvement
 * **[Daniel Schneider](https://github.com/danielsc)**
    * FastAI notebook
+* <b>Evgenia Chroni, Alexandros Ioannou, Kaisar Mussalim</b> 
```

Comment:
can you guys please follow the same format as the other authors?

Reply:
Yes sure. Given that we worked as a team, do you want to add our names separately or it is ok to add our names all together in one line ?

Reply:
Yeah, please add your names separately. Other people, even though as well worked together, had one entry with their name in alphabetical order.